docker-compose.override.yml

```diff
@@ -28,7 +27,11 @@ services:
       - 3000:3000
       - 3035:3035
 
-  queue:
+  assets:
```

Comment:
@roschaefer Can we add the `assets` service to the dependencies of the `app` service so it automatically starts if you run `docker-compose up app`?

Reply:
@roschaefer I rebased and added this change. Please merge if you’re okay with that! :)